src/kademlia/routing_table.cpp

```diff
@@ -77,12 +68,43 @@ void erase_one(T& container, K const& key)
 	container.erase(i);
 }
 
-routing_table::routing_table(node_id const& id, int bucket_size
+void ip_set::insert(address addr)
+{
+	if (addr.is_v4())
+		m_ip4s.insert(addr.to_v4().to_bytes());
+#if TORRENT_USE_IPV6
```

Comment:
I like to re-order these blocks, so the if-statement can be excluded when built without IPv6 support